src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -91,16 +100,12 @@ def __init__(self, net_params, batch_size, num_classes):
         self.predictions = softmax_out
 
         # Loss
-        self.cost = _tf.losses.softmax_cross_entropy(logits=out, onehot_labels=self.one_hot_labels, reduction=_tf.losses.Reduction.NONE)
+        self.cost = _tf.losses.softmax_cross_entropy(logits=out, onehot_labels=one_hot_labels, weights=reshaped_weights, reduction=_tf.losses.Reduction.NONE)
```

Comment:
if you can use pep8 format, that will be better for reviewing purposes.

Reply:
Will do